elements/pfe-styles/src/pf-vars-px.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../pfe-sass/pfe-sass";
+
+// Print t-shirt sizes with pixel units
+:root {
+   --pf-global--FontSize--6xl:  #{$pf-global--FontSize--6xl};
```

Comment:
This would break the theming layer potentially wouldn't it?  Can you help me with when a project would want to use this file?

Reply:
It would change the fallback values from rems to pixels. the global variables are the last one before the actual fallback, so any other variable would still have priority over these. 
`  font-size: var(--pfe-cta--FontSize, var(--pf-global--FontSize--5xl, 2.5rem));`
We are actually overriding these global font size vars in the redhat-theme because of our root font size being 18px on redhat.com, so all typography units based in rems would be too large.